js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
Absolutely, I can create a new repository for the shared library.  What would we like to call it?